src/HealthChecks.InfluxDB/InfluxDBHealthCheck.cs

```diff
@@ -0,0 +1,47 @@
+using InfluxDB.Client;
+using InfluxDB.Client.Api.Domain;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace HealthChecks.InfluxDB
+{
+    public class InfluxDBHealthCheck
+        : IHealthCheck
+    {
+        private readonly InfluxDBClient _influxdb_client;

```

Comment:
`InfluxDBClient` implements `IDisposable` so health check should implement too and call `__influxdb_client.Dispose()` if it owns that object.